test/sandbox/test/view_component_system_test.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ViewComponentSystemTest < ViewComponent::SystemTestCase
+  driven_by :cuprite
+
+  def test_simple_js_interaction_in_browser_without_layout
+    inline_component = render_inline(SimpleJavascriptInteractionWithJsIncludedComponent.new)
+
+    with_inline_rendered_component_path(inline_component) do |path|
+      visit path
+
+      assert find("[data-hidden-field]", visible: false)
+      find("[data-button]", text: "Click Me To Reveal Something Cool").click
+      assert find("[data-hidden-field]", visible: true)
+    end
```

Comment:
🤔 this look good to me, but I'm wondering if we might want to recommend nesting `render_inline` inside the helper call instead of assigning it to `inline_component`? This feels a tad verbose.

I also wonder about calling this `with_rendered_fragment_path`. What do you think of that name?

Reply:
Let's do `with_rendered_component_path` 👍🏻